HOWTO_android.md

```diff
@@ -0,0 +1,637 @@
+<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable MD041 -->
+<p align="left"><img src="https://vulkan.lunarg.com/img/NewLunarGLogoBlack.png" alt="LunarG" width=263 height=113 /></p>
```

Comment:
I don't think we want HTML tags in the Markdown.

Reply:
This same line appears in each of our Markdowns to put the LunarG logo at the top.  Are you sure you want me to remove it?

Reply:
No!  Thanks for pushing that back on me!  Yes, leave it in.  Thank you.